### PR TITLE
Updating Environment Control

### PR DESCRIPTION
**Pull request by gwaygenomics**
Here I add R package versioning from checkpoint and update conda environment and associated documentation.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+channels:
+- r
+- bioconda
 dependencies:
+- r-checkpoint=0.4.0
 - jupyter=1.0.0
 - numpy=1.11.2
-- pandas=0.18.0
-- scikit-learn=0.18.0
+- pandas=0.19.2
```
Comment (dhimmel) [inline environment.yml]: Consider upgrading pandas to `0.20`, which had some major new features but also some breaking changes. Only if you can easily test nothing breaks, but you'll want to be using pandas 0.20+ going forward.

File: environment.yml [hunk 2]
```diff
@@ -1,10 +1,14 @@
 name: pancancer-classifier
+channels:
+- r
+- bioconda
 dependencies:
+- r-checkpoint=0.4.0
```
Comment (dhimmel) [inline environment.yml]: I now specify channels for each package in environment. For example:

```yml
- r::r-checkpoint=0.4.0
- anaconda::jupyter=1.0.0
```

I find it makes it much easier to see where these things are coming from.

File: scripts/install.R [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "readr",
+  "reshape2",
+  "tibble"
+)
+
+pacman::p_load(cran_packages, install = FALSE, character.only = TRUE)
+
+sink("sessionInfo.txt")
+sessionInfo()
+sink()
```
Comment (dhimmel) [inline scripts/install.R]: Final newline missing

File: scripts/util/pancancer_util.R [hunk 4]
```diff
@@ -7,6 +7,8 @@
 #
 # Usage: sourced by scripts using function
 
+checkpoint::checkpoint("2017-06-01", checkpointLocation = ".")
```
Comment (dhimmel) [inline scripts/util/pancancer_util.R]: FYI, I'm now using conda to manage R dependencies without issues (assuming there are no packages not in conda). Something to consider for the future.

File: sessionInfo.txt [hunk 5]
```diff
@@ -0,0 +1,26 @@
+R version 3.3.2 (2016-10-31)
```
Comment (dhimmel) [inline sessionInfo.txt]: This file is very handy for future debugging!

File: environment.yml (hunk 2 above)
Comment (gwaygenomics) [inline environment.yml]: yeah, that is nice - I will update

File: scripts/install.R (hunk 3 above)
Comment (gwaygenomics) [inline scripts/install.R]: arghhhh

File: scripts/util/pancancer_util.R (hunk 4 above)
Comment (gwaygenomics) [inline scripts/util/pancancer_util.R]: yeah, agree it is the way of the future

**Pull request merged**